pkg/pluginmanager/manager.go

```diff
@@ -140,6 +140,8 @@ func DiscoverStandalonePlugins(criteria *discovery.PluginDiscoveryCriteria) ([]d
 	discoveries, err := getPluginDiscoveries()
 	if err != nil {
 		return nil, err
+	} else if len(discoveries) == 0 {
+		return nil, errors.New("there are no plugin discovery sources available. Please run 'tanzu plugin source init' to initialize the discovery source with its default value")
```

Comment:
nit: consider dropping 'with its default value'. Maybe even consider dropping  'to initialize the discovery source with its default value', as it is of that much value to the user.

Reply:
preexisting: but uncovered in the output shown after the the fix:
`there was an error while discovering standalone plugins, error information: ....`
just thinking if we intend to continue to show this message if it should be updated to more accurately reflect what is happening. It is not technically wrong, but the term `standalone` sticks out.

cc @marckhouzam

Reply:
As such there is no error, CLI could not able to find any plugins because there were no discovery sources available.
How about below message:
"there are no plugin discovery sources available. Please run 'tanzu plugin source init'"
OR
"there are no plugin discovery sources available. Please run 'tanzu plugin source init' to initialize plugin discovery source"
@marckhouzam @vuil